x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -355,7 +355,7 @@ func (k EpochCuCache) updateProvidersComplainerCU(ctx sdk.Context, unresponsiveP
 	for _, unresponsiveProvider := range unresponsiveProviders {
 		found := false
 		for _, provider := range pairedProviders {
-			if provider.Address == unresponsiveProvider.Address {
```

Comment:
dont we need to check that the operator is the one that sent the tx?

Reply:
the pairedProviders list is from the ValidatePairingForClient output which constructs it from operator addresses. I can add the check as a sanity check, but it's redundant